debug_toolbar/panels/history/panel.py

```diff
@@ -21,6 +21,22 @@ class HistoryPanel(Panel):
     nav_title = _("History")
     template = "debug_toolbar/panels/history.html"
 
+    def process_request(self, request):
+        response = super().process_request(request)
+        if not self.toolbar.should_render_panels():
+            self.toolbar.store()
+            store_id = self.toolbar.store_id
+            response["DJ-TOOLBAR-BASE-URL"] = (
+                reverse("djdt:render_base") + f"?store_id={store_id}"
+            )
+            sig = SignedDataForm(
+                initial=HistoryStoreForm(initial={"store_id": store_id}).initial
+            ).initial.get("signed")
+            response["dj-history-sidebar-url"] = reverse("djdt:history_sidebar")
```

Comment:
Since this isn't likely to change between requests, this could be moved out of the per-request cycle and into the template as a [JS variable](https://docs.djangoproject.com/en/4.0/ref/templates/builtins/#json-script).

Reply:
:+1: